src/main/java/com/launchacademy/giantleap/models/Bar.java

```diff
@@ -0,0 +1,51 @@
+package com.launchacademy.giantleap.models;
+
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Table(name="bars")
+@Getter
+@Setter
+@NoArgsConstructor
+
+public class Bar {
+  @Id
+  @SequenceGenerator(name="bar_generator", sequenceName="bars_id_seq", allocationSize = 1)
+  @GeneratedValue(strategy= GenerationType.SEQUENCE, generator="bar_generator")
+  @Column(name="id", nullable=false, unique=true)
+  private Integer id;
+
+  @Column(name="bar_name")
+  private String barName;
+
```

Comment:
You should match constraints in the entity and the table. This is NOT NULL in your table, so that should be reflected here.